pkg/arvo/gen/hood/story-write.hoon

```diff
@@ -0,0 +1,31 @@
+::  story: Attach a commit message to the commit associated with a given case on a given desk
+::
+::::
+  ::
+/-  *story
+:-  %say
+|=  $:  [now=@da eny=@uvJ bec=beak]
+        [[=prose ~] =desk cas=cash ~]
```

Comment:
I would explode this to `[title=@t body=@t ~]` so that you can just run `|story-write 'title' 'body'` instead of needing brackets

Reply:
And then to `[title=@t body=$@(~ [p=@t ~])]` so you can have just a title \:P
(Also I wonder what a snappier for this, most common, command would be - possibly just `|story`?)

Reply:
> And then to `[title=@t body=$@(~ [p=@t ~])]` so you can have just a title :P

what’s the advantage of using that type vs. a unit?

Reply:
In the "body is present" case you don't need a sig prior to the body

Reply:
Yeah it's a "list with one or two elements" type
which maps more cleanly to dojo generator syntax

Reply:
that clears it up, thx

Reply:
done in https://github.com/urbit/urbit/pull/5612/commits/18537e0ae0820d2f1a93bacb25f8ee6fa28ad59f